backend/proto/message.proto

```diff
@@ -0,0 +1,21 @@
+package  launchpadpackage;
+syntax = "proto3";
+
+message Worker {
```

Comment:
Proto definition should be like `Type snake_name = <field_id>`

Reply:
fixed